sdk/include/game-contract-sdk/game_base.hpp

```diff
@@ -16,6 +15,8 @@
 #include <game-contract-sdk/rsa.hpp>
 #include <game-contract-sdk/dispatcher.hpp>
 
+#include "service.hpp"
```

Comment:
better use '<>' notation to include

Reply:
Why? `<>` for global imports and `""` for local. If we always use `<>` - need to append it to code style (and create it)

Reply:
Just for convenience, explicit definition of project and sub-project in include path make more simpler to reading code and also it helps to write include paths in cmake files.

Reply:
basically, some guides says that need to define full path related to project base in quotes, but in that case `<>`  is more likely :)